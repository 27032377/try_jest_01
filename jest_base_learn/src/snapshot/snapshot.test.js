const { generateProps } = require('./generateProps');

it('Snapshot快照测试', () => {
    expect(generateProps()).toMatchSnapshot();
})