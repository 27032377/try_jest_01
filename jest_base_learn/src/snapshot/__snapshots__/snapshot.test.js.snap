// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot快照测试 1`] = `
Object {
  "desc": "测试",
  "name": "jest",
  "onChange": [Function],
  "time": "2020",
}
`;
